components/pkg-export-kubernetes/defaults/KubernetesManifest.hbs

```diff
@@ -40,3 +40,6 @@ spec:
     ## encrypts the communication between Habitat supervisors.
     ringSecretName: {{ring_secret_name}}
 {{/if}}
+{{#if bind}}
+    bind:
```

Comment:
Should there be a value on the right side?

Reply:
No. The properties of binds are specified on the lines below here and I added a separate template for those (KubernetesBind.hbs) since there can be multiple binds.